docker-for-azure/faqs.md

```diff
@@ -69,3 +69,34 @@ $ sudo ping 10.0.0.4
 ```
 
 Note that access to Docker for Azure and Azure happens through a shell container that itself runs on Docker.
+
+
+### What are the different Azure Regions?
+All regions can be found here: https://azure.microsoft.com/en-us/regions/
+An exerpt of the above regions to use when you create your service principal are:
+```
```

Comment:
You need a newline after line 76.

Do we really want to list out all these regions? What if they change? Isn't it sufficient to have the link? (btw you need to linkify the link, that doesn't happen automatically) and maybe an example specifying a region?